pom.xml

```diff
@@ -78,6 +78,11 @@
             <artifactId>splunk</artifactId>
             <version>1.6.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>9.1-901-1.jdbc4</version>
```

Comment:
Is there any reason why such an old version of the Postgres driver is used?
The latest version available in maven is 42.2.1.
https://mvnrepository.com/artifact/org.postgresql/postgresql

Reply:
My bad. I thought that was the latest.

Reply:
Np